client/src/styles/PortfolioTable.scss

```diff
@@ -107,7 +107,13 @@ $table-border-dark: 1px solid $dark;
           border: 1px solid $justfix-grey-700;
         }
         .contacts-list {
-          color: $justfix-green;
+          text-align: left;
+          position: relative;
+          list-style-type: none;
+          left: 130.5rem;
```

Comment:
yeah definitely agree! I am imagining once design gets to this it might look completely different (and maybe not even involve the expanded row), so this should just be temporary.